### ISSUE TITLE
Replace exe icon via command line

### ISSUE THREAD

**Issue by hisune** [nwutils/nw-builder#305]
I try this:

```
nwbuild -p win32 -v 0.12.3 --winIco E:\build\resource\icon.ico -o E:\build\target E:\build\client 
```

The `E:\build\resource\icon.ico` file is a 48 x 48 ico file.
But it still show like that:
![ss](https://cloud.githubusercontent.com/assets/7941669/14595577/ed2e3d92-0572-11e6-845e-ac349ec99590.jpg)

And in Resource Hacker, it has two icon:
![ss1](https://cloud.githubusercontent.com/assets/7941669/14595608/34c06cca-0573-11e6-9300-5c14e8546879.jpg)

![ss2](https://cloud.githubusercontent.com/assets/7941669/14595611/3826da34-0573-11e6-8308-00ce23577672.jpg)

Why?

**Comment by adam-lynch** [comment]
This could be a bug. We're going to look into improving this area soon.

**Comment by cumtRobin** [comment]
any progress?

**Comment by Blatman** [comment]
Make sure that the ico file specified is a proper windows ico file with a 256x256 size included - winresourcer usually sticks the 256 size into the first icon slot and the app will use that.

**Comment by ayushmxn** [comment]
Fixed by #566 using the `--useRcedit` option.
